### ISSUE TITLE
Problem display dates on form and tabulator component

### ISSUE THREAD

**Issue by chavu** [rjaros/kvision#334]
I'm having a challenge displaying dates on forms and tables.
1. On a form, when l use myForm.setData(myEntity),  to edit date in the form, the date field value is always reverting the current system date ignoring the value in the object
2.  On the TabulatorRemote table,  dates are reverting to system date when I use  formatter below:

```
formatterComponentFunction = { _, _, item  ->
  span( item.startDatePlanned?.toStringF("DD-MM-YYYY"))
}
```

**Comment by rjaros** [comment]
It should work fine. Please check this sample application and compare your code:
```kotlin
@Serializable
data class MyForm(@Contextual val mydate: Date)

class App : Application() {

    override fun start() {
        root("kvapp") {
            vPanel(spacing = 20) {
                margin = 20.px
                width = 400.px
                val form = formPanel<MyForm> {
                    dateTime(label = "My date:", format = "DD-MM-YYYY").bind(MyForm::mydate)
                }
                button("Set the date").onClick {
                    val date = "13-04-2017".toDateF("DD-MM-YYYY")
                    form.setData(MyForm(date))
                }
                button("Get the date").onClick {
                    console.log(form.getData().mydate)
                }
            }
        }
    }
}

fun main() {
    startApplication(
        ::App,
        module.hot,
        BootstrapModule,
        BootstrapCssModule,
        BootstrapDatetimeModule,
        FontAwesomeModule,
        CoreModule
    )
}

```

**Comment by rjaros** [comment]
On the other hand there is indeed some issue with `TabulatorRemote`. I'm investigating this.

**Comment by rjaros** [comment]
The problem was different date/time serialization format used by KVision fullstack interfaces. I've changed the `TabulatorRemote` component to use correct `DateSerializer`. The fix will be available in the next release.

**Comment by chavu** [comment]
I'm happy you resolved this. How can I get the fix before the next release because I want to deploy this app I'm working on in the next 7days. This issue is also the reason the edit form is showing the current date because I'm getting the data from the tabulator row object.

**Comment by rjaros** [comment]
I plan to publish 5.7.2 this weekend. Stay tuned.

**Comment by chavu** [comment]
Great. Thank you.

**Comment by rjaros** [comment]
Released with 5.7.2
